_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,135 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each `<meta name="viewport">` element with a `content` attribute.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- either does not have the property `user-scalable` or has it set to yes; and
+- does not have the property `maximum-scale`. 
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+- There may be some inconsistencies with how different browser vendors interpret `<meta name="viewport">`.
+
+## Background
+
+- [WCAG Success Criterion 1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow)
```

Comment:
You don't need to link to WCAG. The accessibility requirements mapping has that link.

Reply:
This is marked as resolved, but the problematic link is still there? 
Makes it a bit hard to find out if I am reviewing an old version of the rule???